lib/api_local.ml

```diff
@@ -164,6 +170,21 @@ module Slack_json : Api.Slack = struct
 end
 
 module Buildkite : Api.Buildkite = struct
+  let get_job_log ~ctx:_ (job : Buildkite_t.job) =
+    match job.log_url with
+    | None -> Lwt.return_error "Unable to get job log, job has no log_url field"
+    | Some log_url ->
+    match Re2.find_submatches_exn Util.Build.buildkite_api_org_pipeline_build_job_re log_url with
+    | exception _ -> failwith "Failed to parse Buildkite build url"
```

Comment:
```suggestion
    | exception exn -> Exn.fail ~exn  "failed to parse buildkite build url %s" log_url
```

without details this message can't be leveraged in production as we won't know what has failed

Reply:
done